CHANGELOG.md

```diff
@@ -550,3 +572,160 @@ The following rules are now recommended:
   ```js
   const using = resource.lock();
   ```
+
+[noAccessKey]: https://biomejs.dev/lint/rules/noAccessKey/
```

Comment:
I think we should update our guidelines if we want to start to use this feature. In the guidelines we should explain how links are added.

I'm not a big fan of this, because it forces people to add a link at the bottom of the document, which will grow at every release. Plus it forces us to search if a link already exists.

Reply:
I share your opinion about labeled links. However, we only have to add new rules. All existing rules are already there.

By the way, I can reverse this change.